helix-view/src/editor.rs

```diff
@@ -1406,13 +1409,14 @@ impl Editor {
         doc_id: DocumentId,
         path: Option<P>,
         force: bool,
+        config: &Config,
```

Comment:
Removing the explicit argument from `Document::save` as suggested above also removes the need for this argument solving your concern :)